docs/aws/GettingStarted/DeviceCredentials.rst

```diff
@@ -20,7 +20,7 @@ Generate a CA certificate
 Creating the CA certificate is a one-time operation.
 If you have a directory called :file:`certificates` with a :file:`rootCA.pem` file in it, you have already completed this step.
 
-It is recommended to use your `own Certificate Authority (CA) <https://docs.aws.amazon.com/iot/latest/developerguide/device-certs-your-own.html>`_ to create certificates for your devices since it allows generating device certificates offline.
+ (`This is the recommended approach <https://docs.aws.amazon.com/iot/latest/developerguide/device-certs-your-own.html>`_) and allows generating device certificates offline.
```

Comment:
Does this really need to change? It looks like the meaning is the same, but now the fact of using your own Certificate Authority is hidden behind the link.

Reply:
Yes, I think this improves it. Because previously the sentence gave the impression that the user needs to to this in addition when they go to production, but actually this is the way it is implemented in nRF Asset Tracker.

Reply:
I would still prefer something other than "this", since the reader has to click through the link to actually see what "this" is.

Reply:
```suggestion
 (`Using a custom Certificate Authority (CA) is the recommended approach <https://docs.aws.amazon.com/iot/latest/developerguide/device-certs-your-own.html>`_) and allows generating device certificates offline.
```

How about this?

Reply:
```suggestion
 nRF Asset Tracker for AWS follows the recommendation of (`using a custom Certificate Authority (CA) <https://docs.aws.amazon.com/iot/latest/developerguide/device-certs-your-own.html>`_) which allows generating device certificates offline.
```

Or this?

Reply:
The second one is really good. Clear, and gets all the information.